js/getCountryCode.js

```diff
@@ -0,0 +1,260 @@
+var ffQSprint = {
+	getCountryCode: function (countryName) {
+	// if we don't have the country in our list, we use the world icon directly
```

Comment:
please indent this correctly, this should be a level further in. Also note that we are using 2 spaces per level.